src/schema/v2/fields/pagination.ts

```diff
@@ -217,6 +221,7 @@ export const paginationResolver = <T>({
     ...connectionFromArraySlice(body, connectionArgs, {
       arrayLength: totalCount,
       sliceStart: offset,
+      resolveNode,
```

Comment:
Also wondering why this feature needed another API addition?

Reply:
Yep, this is just another one of the options for `connectionFromArraySlice` that we have to thread through. We've just never had to create a custom edge with the `paginationResolver` before.

Reply:
👍